.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "framework"]
 	path = framework
-	url = git@github.com:venediktov/vanilla-rtb.git
+	url = https://github.com/venediktov/vanilla-rtb.git
+[submodule "submodules/redisclient"]
+	path = submodules/redisclient
+	url = https://github.com/nekipelov/redisclient
```

Comment:
url = https://github.com/nekipelov/redisclient.git

You are missing extension .git  , that's probably why it does not load it in travis ?

CMake Error: The source directory "/home/travis/build/vanilla-rtb/rapid-bidder/build-vanilla/submodules/redisclient" does not appear to contain CMakeLists.txt.
Specify --help for usage, or press the help button on the CMake GUI.
make[2]: *** [submodules/redisclient/src/redisclient-stamp/redisclient-configure] Error 1
make[2]: Leaving directory `/home/travis/build/vanilla-rtb/rapid-bidder/build-vanilla'
make[1]: *** [CMakeFiles/redisclient.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....